scripts/fossa.js

```diff
@@ -0,0 +1,49 @@
+const util = require("node:util");
```

Comment:
is this script ever intended to be run outside of the GH Pipeline? we may want to add a `node` dependency to our hermit env or else convert this to a Bash shell script

no big deal really, I'm cool with it as is, but it's an idea

Reply:
Ah Nice catch, this file was a mistake!